playbooks/agent_driver_deploy.yaml

```diff
@@ -0,0 +1,44 @@
+---
+# version: 2016-12-29
+- hosts: all
+  vars_files:
+    - vars/agent_driver_deploy_vars.yaml
+  become_user: root
+  become_method: sudo
+  become: true
+  remote_user: '{{ remote_user }}'
+  roles:
```

Comment:
Updating to add new playbooks for driver deployment, then a separate one for the agent deployment.